pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.33</version>
+    <version>2.35</version>

```

Comment:
3.2 would be preferable if you are updating the parent pom version.

Reply:
https://issues.jenkins-ci.org/browse/JENKINS-45245

Reply:
Right, I understand the issue. I am just saying, if upgrading the parent pom version, upgrade to something more recent like 3.2 so it's more inline with recent changes.

Reply:
gotcha. I was just trying to introduce "minimal" amount of changes

Reply:
may be I can also bump `jenkins.version` then?

Reply:
Let's leave that for now, that's a bigger change (it would require that version of Jenkins to use the plugin). I am considering removing the jenkins.version property to pick up the default from the parent pom, but still haven't decided yet.